### ISSUE TITLE
proxyHost and proxyHttps options do not work.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [GoogleCloudPlatform/appengine-maven-plugin#46]
```
Using the following configuration options:
<proxyHost>proxy.xxx.com:80</proxyHost> 

produces the following output:

...
[INFO] Retrieving Google App Engine Java SDK from Maven
[INFO] Updating Google App Engine Application
Bad argument: Unknown option: --proxy
usage: AppCfg [options] <action> [<app-dir>] [<argument>]
...

same thing with proxyHttps, which produces:
Bad argument: Unknown option: --proxy_https

Both error messages look weird as later in the output both --proxy and 
--proxy_https are listed as acceptable options.

I get this error launching maven both from the command line and from within 
eclipse.

I tried setting jvm arguments to force the use of a proxy, but it seems that 
appcfg is launched in an separate jvm than the one running maven and I couldn't 
find a way to pass jvm options as for many plugins with a similar design.

Bottom line I didn't find a way to use the plugin from behind my corporate 
proxy.

Also as a separate note I didn't find any page with detailed documentation on 
the options available for this plugin.
These pages:
https://developers.google.com/appengine/docs/java/tools/maven
https://code.google.com/p/appengine-maven-plugin/

are good for a start but incomplete.

Thanks
Raffaele


```

Original issue reported on code.google.com by `raffaele...@gmail.com` on 6 Dec 2013 at 3:11
